pages/admin/dashboard.js

```diff
@@ -0,0 +1,229 @@
+import { useRouter } from "next/router";
+import { useEffect, useState } from "react";
+import { CircularProgress, Typography } from "@material-ui/core";
+import PropTypes, { array } from "prop-types";
+import styles from "../../styles/GameRow.module.css";
+import game from "../../helpers/game";
+
+import firebase from "../../utils/firebase";
+import AdminGameView from "./game";
+
+import Link from "next/link";
+import {
+  Box,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  CardHeader,
+  Container,
+  IconButton,
+  Paper,
+  Snackbar,
+  TextField,
+} from "@material-ui/core";
+
+export default function Dashboard() {
+  const [gameArrayState, setGameArrayState] = useState([]);
+  const [gameClick, setGameClick] = useState(false);
+  const [login, setLogin] = useState(false);
+  const [gamePin, setGamePin] = useState();
+
+  const router = useRouter();
+
+  if (firebase.apps.length === 0) {
+    console.log("not initialized");
+  } else {
+    firebase.auth().onAuthStateChanged(function (user) {
+      if (user) {
```

Comment:
I know you're still working on this.

Since we're using anonymous users, everyone will be logged in (if `user` will be true). So we'll need to check if their email domain is wafbla.org (as you had suggested)